### PR TITLE
simplify shell echo sample

### PR DESCRIPTION
**Pull request by jldec**
```
~/riff/riff/samples/shell/echo (simpler-echo)$ riff publish -d world -r 
Posting to http://192.168.99.100:31982/requests/echo
hello
world
```

### CODE REVIEW

File: samples/shell/echo/echo.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-printf "received at `date -u`: `cat`"
+echo hello ; cat
```
Comment (markfisher) [inline samples/shell/echo/echo.sh]: how about this? `xargs echo hello`

Comment (glyn) [reply samples/shell/echo/echo.sh]: +1: no gratuitous newline.

**Comment by jldec** [comment]
@glyn i took @markfisher's suggestion and added a "hello" sample. The echo sample just has `cat` now. What do you think?

**Comment by trisberg** [comment]
Squashed and merged as commit ce2d769cbc18c4c5c469f1b5e91db5684c77273c

**Comment by glyn** [comment]
@jldec `cat` LGTM, but you'll probably need to give a brief explanation in demos as not everyone will know that `cat` reads from standard input by default. (I had to look it up, for example.)

**Pull request closed without merging**